SetDefaultSinkId/explainer.md

```diff
@@ -0,0 +1,185 @@
+# Set Default Audio Output Device
```

Comment:
Should we also add an entry to this explainer to the top level readme.md?

For example, here's the entry for the media iframe pausing explainer we're publishing:

https://github.com/MicrosoftEdge/MSEdgeExplainers/pull/770/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

Reply:
thanks, updated.